app/models/agenda.js

```diff
@@ -4,6 +4,7 @@ export default class Agenda extends Model {
   @attr agendaStatus;
   @attr agendaType;
   @attr renderedContent;
+  @attr('boolean', { defaultValue: false }) deleted;
```

Comment:
why does an agenda have a deleted status?

Reply:
Because the whole thing is a mess, the versionedAgenda is not linked to published/signed resources and is this one who it is

Reply:
:+1: